Documentation/templates/strings/objc.md

```diff
@@ -0,0 +1,53 @@
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File names | strings/objc-string-h.stencil & strings/objc-string-m.stencil |
+| Configuration example | <pre>strings:<br />  inputs: path/to/Localizable.strings<br />  outputs:<br />    templateName: objc-string-h<br />    output: Localizable.h</pre><br />    templateName: objc-string-m<br />    output: Localizable.m</pre> |
```

Comment:
There's something wrong here, also `outpus` is an array, so there should be a `- ` before each `templateName` here:
```yaml
strings:
  inputs: path/to/Localizable.strings
  outputs:
    - templateName: objc-string-h
      output: Localizable.h
    - templateName: objc-string-m
      output: Localizable.m
```

Reply:
I'll add '- ' before each 'templateName', and '  ' before each 'output'.  

Is there something else?